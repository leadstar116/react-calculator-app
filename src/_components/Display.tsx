import React from 'react'

type Props = {
  value: string,
}

const Display = (props: Props) => {
  return (
    <div className="calculator-display">
      <div>{props.value}</div>
    </div>
  );
}

export default Display