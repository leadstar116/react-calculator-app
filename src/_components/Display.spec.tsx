import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Display from './Display'

describe('Display Component', () => {

  let component = renderer.create(
    <BrowserRouter>
      <Display value={"0"} />
    </BrowserRouter>
  );

  it('should render with given props', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
})