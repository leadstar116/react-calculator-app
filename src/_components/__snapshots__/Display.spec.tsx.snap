// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Component should render with given props 1`] = `
<div
  className="calculator-display"
>
  <div>
    0
  </div>
</div>
`;
